tt_metal/hw/inc/firmware_common.h

```diff
@@ -66,6 +44,19 @@ inline void do_crt1(uint32_t tt_l1_ptr* data_image) {
     }
 }
 
+void noc_bank_table_init(uint64_t mem_bank_to_noc_addr)
```

Comment:
hm, I think I deleted the wrong comment of mine.  Anyway this either needs to be marked inline, or moved to substitutions.cpp for the same reason `l1_to_local_mem_copy` was moved.